docs/src/pages/components/searchfield/demo.tsx

```diff
@@ -1,17 +1,38 @@
 import * as React from 'react';
+import { SearchField } from '@aws-amplify/ui-react';
+import { useSearchFieldProps } from './useSearchFieldProps';
+import { SearchFieldPropControls } from './SearchFieldPropControls';
+import { Demo } from '@/components/Demo';
 
-import { Flex, SearchField, View } from '@aws-amplify/ui-react';
-
-import { Example } from '@/components/Example';
-import { useSearchFieldProps } from '@/components/useSearchFieldProps';
-import { SearchFieldPropControls } from '@/components/SearchFieldPropControls';
+const propsToCode = (searchFieldProps) => {
+  return (
+    `<SearchField` +
```

Comment:
This is organized really well. Me and @dbanksdesign are looking for a better way to do so #1172, feel free to share your ideas if you have any.